pandaharvester/harvestercore/plugin_factory.py

```diff
@@ -49,6 +49,6 @@ def get_plugin(self, plugin_conf):
         impl = cls(**args)
         # bare instance when middleware is used
         if 'original_config' in plugin_conf:
-            bare_impl = self.get_plugin(plugin_conf['original_config'])
+            bare_impl = plugin_conf['original_config']
```

Comment:
I think that the following line

https://github.com/PanDAWMS/panda-harvester/blob/master/pandaharvester/harvestermiddleware/rpc_herder.py#L27

needs to be changed accordingly if the dict is set to self.bare_impl.

Reply:
solved